easybuild/easyconfigs/e/ENT/ENT-1.0.2.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Binary"
```

Comment:
@rtripath89 You'll want to use `BinariesTarball` instead, since `Binary` doesn't unpack the `sources`...

Did you test this easyconfig?